src/components/Slider/components/Slider.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+
+class Slider extends React.Component {
```

Comment:
import Component from react so you can write Slider extends Component

Reply:
What's the difference?

Reply:
there's no difference in the execution, but we use object destructuring everywhere(almost) we can
so the code is easier to read and mantain,